lib/rules/component-tags-order.js

```diff
@@ -25,6 +26,10 @@ module.exports = {
       url: 'https://eslint.vuejs.org/rules/component-tags-order.html'
     },
     fixable: 'code',
+    messages: {
+      unexpected:
```

Comment:
Is there a reason to move the "message"? If not, revart the position to minimize the difference.

Reply:
Sorry about that. I think what happened is that my text editor alphabetized the keys on save. I agree we should try to keep the PR as small as possible, so I've turned off format on save for this workspace because it was causing this kind of trouble in my PRs.